cpp/src/rolling/rolling_detail.hpp

```diff
@@ -28,25 +29,42 @@ namespace detail
 {
   // return true if ColumnType is arithmetic type or
   // AggOp is min_op/max_op/count_op for wrapper (non-arithmetic) types
-  template <typename ColumnType, class AggOp, bool is_mean>
+  template <typename ColumnType, class AggOp, cudf::experimental::aggregation::Kind op, bool is_mean>
```

Comment:
It seems like the `is_valid_aggregation` function could be used to simplify this logic: https://github.com/rapidsai/cudf/blob/branch-0.12/cpp/include/cudf/detail/aggregation/aggregation.hpp#L298

Reply:
this will not help as we don't support Median, QUANTILE and etc. So, this will give error.

Reply:
Right, you can't use it alone, but you could simplify the logic to something like:
```
is_valid_rolling_agg and is_valid_aggregation<ColumnType, AggOp>;
```
instead of:
```
    return !std::is_same<ColumnType, cudf::string_view>::value &&
           is_valid_rolling_agg &&
           (cudf::is_numeric<ColumnType>() ||
            comparable_countable_op ||
            timestamp_mean);
```